yb-voyager/src/query/queryparser/ddl_processor.go

```diff
@@ -924,6 +924,44 @@ func (mv *MView) GetSchemaName() string { return mv.SchemaName }
 
 func (mv *MView) GetObjectType() string { return MVIEW_OBJECT_TYPE }
 
+// ============================Function Processor =================
+
+type FunctionProcessor struct{}
+
+func NewFunctionProcessor() *FunctionProcessor {
+	return &FunctionProcessor{}
+}
+
+func (mv *FunctionProcessor) Process(parseTree *pg_query.ParseResult) (DDLObject, error) {
+	funcNode, ok := getCreateFuncStmtNode(parseTree)
+	if !ok {
+		return nil, fmt.Errorf("not a CREATE VIEW statement")
+	}
+
+	funcNameList := funcNode.CreateFunctionStmt.GetFuncname()
+	funcSchemaName, funcName := GetFunctionObjectName(funcNameList)
+	function := Function{
+		SchemaName: funcSchemaName,
+		FuncName:   funcName,
+		ReturnType: GetReturnTypeOfFunc(parseTree),
+	}
+	return &function, nil
+}
+
+type Function struct {
+	SchemaName       string
+	FuncName         string
+	ReturnType       string
+	IsReturnPercType bool
```

Comment:
what is this field? doesn't seem to be used?

Reply:
yeah, not using it currently, will remove it.